zerver/lib/test_classes.py

```diff
@@ -70,6 +71,14 @@ def flush_caches_for_testing():
     global API_KEYS
     API_KEYS = {}
 
+def unsign_subdomain_cookie(result):
+    # type: (HttpResponse) -> Dict[str, Any]
+    key = 'subdomain.signature'
+    salt = key + 'zerver.views.auth'
+    cookie = result.cookies.get(key)
+    value = signing.get_cookie_signer(salt=salt).unsign(cookie.value, max_age=15)
+    return ujson.loads(value)
+
```

Comment:
This should go into `test_helpers`, not `test_classes`.  I moved this while merging.